### PR TITLE
[ADDED]Getting started for Resume Screener Frontend

### PR DESCRIPTION
**Pull request by tanishakumar18**

### CODE REVIEW

**Comment by kamalika0363** [comment]
[ADDED] #22 
README for Resume Screener

File: Getting Started.md
Comment (vish-rt) [inline Getting Started.md:1]: The content of this file should be in `README.md` only. Putting this in any other file would not reflect within the repository's description.
- [ ] Copy everything written in this file
- [ ] Replace the contents of `README.md` with this

File: Getting Started.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
```
Comment (vish-rt) [inline Getting Started.md:1]: While writing a project's description, always begin with the project's name. Leverage heading tags (#) to highlight them.

**Review by vish-rt — CHANGES_REQUESTED** [review]
Recommend having a look at the suggestions and make new commits to your fork.

File: Getting Started.md (hunk 1 above)
Comment (tanishakumar18) [reply Getting Started.md:1]: Only the name of the project has to be added at the top?

**Review by vish-rt — APPROVED** [review]
Great!

**Pull request merged**